test/syntax-errors.json

```
@@ -17,6 +16,10 @@
   "missing space {|foo|:func}",
   "missing space {|foo|@bar}",
   "missing space {:func@bar}",
+  "missing space {:func @bar@baz}",
+  "missing space {:func @bar=42@baz}",
+  "missing space {+reserved@bar}",
+  "missing space {&private@bar}",
```

These don't seem like "missing space".
```suggestion
  "missing equals {:func @bar@baz}",
  "missing missing quotes {:func @bar=42@baz}",
  "reserved {+reserved@bar}",
  "private {&private@bar}",
```

---

All of those can be turned into valid syntax with the addition of a single space:
```suggestion
  "missing space {:func @bar @baz}",
  "missing space {:func @bar=42 @baz}",
  "missing space {+reserved @bar}",
  "missing space {&private @bar}",
```
(the above is not intended as an actual suggestion, just to show a nice diff)

---

Technically, though, they are invalid name tests. You created them by removing non-optional spaces, of course.